packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1107,6 +1107,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         calloutWidth={useComboBoxAsMenuWidth && this._comboBoxWrapper.current ? comboBoxMenuWidth && comboBoxMenuWidth : dropdownWidth}
         calloutMaxWidth={dropdownMaxWidth ? dropdownMaxWidth : comboBoxMenuWidth}
         hidden={persistMenu ? !isOpen : undefined}
+        shouldRestoreFocus={this.props.shouldRestoreFocus}
```

Comment:
This should probably default to true as well, I think at least.

Reply:
Because it should be restoring focus, unless otherwise noted I mean. So if the underlying component changes, it's still passing in true. That's probably overkill though.